src/Compiler/AbstractIL/il.fs

```diff
@@ -3902,7 +3902,7 @@ let prependInstrsToCode (instrs: ILInstr list) (c2: ILCode) =
     // If there is a sequence point as the first instruction then keep it at the front
     | I_seqpoint _ as i0 ->
         let labels =
-            let dict = Dictionary.newWithSize c2.Labels.Count
+            let dict = Dictionary.newWithSize (c2.Labels.Count * 2)
```

Comment:
It means that dictionaries have lower chance of collisions when they are not operating at their peak capacity.
E.g. the last item had a collision chance of (N-1)/(N), whereas after the chance the collision chance is reduced to half.

I will add a small comment so that reader knows what to google at least.

Reply:
Alright, that's good enough